turingschool/savile
# V1.1

#### What does this PR do?
This PR makes a MINOR update to v1.0 of Savile, bringing it to v1.1. The change is, adding a declaration for the `font-family` on the base button styles.

#### Where should the reviewer start?

- `css/design_system/elements/button.scss`
- `css/versions/v1` directory - added 1.1 and updated latest to match it

#### How should this be manually tested?

You could run locally and visit http://localhost:4000/css/versions/v1/1.1.css and see that the file now has the `font-family` declaration?

#### Any background context you want to provide?


#### What are the relevant tickets?


#### Screenshots (if appropriate)


#### Any other deploy steps?

After merging this, I will follow Step 8 from the Contribution Guide:
> Once merged make a new release in Github by going to https://github.com/turingschool/savile/releases and click 'Draft a new release'. Tag version (e.g v1.3),target (master) and release title (e.g 1.3), and what new features are included
